### PR TITLE
Added content provider Voot

### PR DESCRIPTION
**Pull request by gakudva**
Voot.com - content provider in India. All media channels owned by Viacom18 Media Private Limited.

As a follow-up to: https://github.com/AdguardTeam/AdGuardHome/pull/5451

### CODE REVIEW

File: assets/services.json [hunk 1]
```diff
@@ -2146,5 +2146,13 @@
       ],
       "icon_svg": "<svg xmlns=\"http://www.w3.org/2000/svg\" fill=\"currentColor\" viewBox=\"0 0 30 30\"><path d=\"M14.46 14.191H9.982c0-.471.033-.954.039-1.458v-5.5h5.106V5.935a1.352 1.352 0 00-.404-.957 1.378 1.378 0 00-.968-.396H5.783c.028-.088.056-.177.084-.255.274-.82 1.153-3.326 1.153-3.326a4.262 4.262 0 00-2.413.698c-.57.4-.912.682-1.371 1.946-.532 1.453-.997 2.856-1.31 3.693C1.444 8.674.28 11.025.28 11.025a5.85 5.85 0 002.52-.61c1.119-.593 1.679-1.502 2.054-2.883l.09-.3h2.334v5.5c0 .5-.045.982-.073 1.46h-4.12c-.71 0-1.39.278-1.893.775a2.638 2.638 0 00-.783 1.874h6.527a17.717 17.717 0 01-.778 3.649 16.796 16.796 0 01-3.012 5.273A33.104 33.104 0 010 28.74s3.13 1.175 5.425-.954c1.388-1.292 2.631-3.814 3.23-5.727a28.09 28.09 0 001.12-5.229h5.967v-1.37a1.254 1.254 0 00-.373-.899 1.279 1.279 0 00-.909-.37zm-3.19 5.484l-2.312 1.491 5.038 7.458a6.905 6.905 0 00.672-2.218 3.15 3.15 0 00-.28-2.168l-3.118-4.563zM29.05 4.582H16.733V25.94h3.018l.403 2.572 4.081-2.572h4.815V4.582zm-5.207 18.69l-2.396 1.509-.235-1.508h-1.724V7.233h6.78v16.04h-2.425z\" /></svg>"
     }
```
Comment (Sergey-Lyapin) [inline assets/services.json:2148]: comma must be here

File: assets/services.json [hunk 2]
```diff
@@ -2146,5 +2146,13 @@
...
...04v32.16c0,28.45-13.4,56.05-62.72,56.05S142.12,318.24,142.12,289.79zM226.53,290.93v-34.44c0-13.09-6.84-20.5-21.65-20.5s-21.65,7.4-21.65,20.5v34.44c0,13.09,6.82,20.49,21.65,20.49C219.7,311.42,226.53,304.02,226.53,290.93z M281.39,289.79v-32.16c0-28.46,13.41-56.04,62.74-56.04s62.72,27.58,62.72,56.04v32.16c0,28.45-13.4,56.05-62.72,56.05S281.39,318.24,281.39,289.79z M365.8,290.93v-34.44c0-13.09-6.86-20.5-21.65-20.5s-21.65,7.4-21.65,20.5v34.44c0,13.09,6.86,20.49,21.65,20.49S365.8,304.02,365.8,290.93zM510.42,335.43l-2.73-17.48l-0.56-2.33c-0.77-2.64-3.19-4.46-5.94-4.47l-2.24,0.34c-3.19,0.52-6.42,0.78-9.65,0.79c-2.37,0.45-4.79-0.5-6.22-2.44c-1.41-3.5-2.2-7.22-2.34-11v-55.65c0.01-3.18,2.59-5.75,5.77-5.76h17.84c3.22-0.05,5.89-2.53,6.16-5.74l1.48-21.53c0.23-2.93-1.95-5.49-4.88-5.72l-0.49-0.02h-20.13c-3.18,0-5.75-2.57-5.77-5.75v-27.49c0.08-2.7-2.05-4.95-4.75-5.03l-0.96,0.07l-29.64,4.12c-3.22,0.53-5.62,3.27-5.71,6.54v21.79c-0.01,3.18-2.59,5.75-5.77,5.75h-7.27c-3.18,0-5.75,2.57-5.77,5.75v21.51c0.01,3.18,2.59,5.75,5.77,5.75h7.27c3.18,0,5.76,2.58,5.77,5.76v67.11c0,25.61,15.4,35.56,41.91,35.56c8.07-0.11,16.1-1.24,23.89-3.36l0.1-0.09c2.88-0.59,4.96-3.12,4.97-6.06l-0.18-0.87L510.42,335.43z\" /></svg>"
```
Comment (Sergey-Lyapin) [inline assets/services.json:2155]: Please use this compressed version of SVG
```
<svg xmlns=\"http://www.w3.org/2000/svg\" fill=\"currentColor\" viewBox=\"0 0 512 512\"><path d=\"M96 340c-1 4-4 6-7 6H48c-3 0-6-2-8-6L0 213c-1-2 0-5 2-5l2-1h30c4 0 7 3 8 6l25 87c1 3 2 3 3 0l25-87c1-3 4-6 7-6h31c2 0 4 2 4 4v2L96 340zm46-50v-32c0-29 14-56 63-56s63 27 63 56v32c0 28-14 56-63 56s-63-28-63-56zm85 1v-35c0-13-7-20-22-20s-22 7-22 20v35c0 13 7 20 22 20s22-7 22-20zm54-1v-32c0-29 14-56 63-56s63 27 63 56v32c0 28-14 56-63 56s-63-28-63-56zm85 1v-35c0-13-7-20-22-20s-21 7-21 20v35c0 13 6 20 21 20s22-7 22-20zm144 44-2-17-1-2c-1-3-3-5-6-5h-2l-10 1c-2 1-4 0-6-2l-2-11v-56c0-3 2-6 6-6h17c4 0 6-2 7-5l1-22c0-3-2-5-5-6h-21c-3 0-5-2-5-5v-28c0-3-2-5-5-5h-1l-30 4c-3 1-5 4-5 7v22c0 3-3 5-6 5h-7c-4 0-6 3-6 6v22c0 3 2 5 6 5h7c3 0 6 3 6 6v67c0 26 15 36 42 36 8 0 16-1 23-4h1c2 0 5-3 5-6l-1-1z\"/></svg>
```

Comment (gakudva) [reply assets/services.json]: made the changes

File: assets/services.json (hunk 1 above)
Comment (gakudva) [reply assets/services.json]: I have added the missing "," before the new addition.  Hope its OK now.

File: assets/services.json [hunk 3]
```diff
@@ -2145,6 +2145,14 @@
         "||zhimg.com^"
       ],
       "icon_svg": "<svg xmlns=\"http://www.w3.org/2000/svg\" fill=\"currentColor\" viewBox=\"0 0 30 30\"><path d=\"M14.46 14.191H9.982c0-.471.033-.954.039-1.458v-5.5h5.106V5.935a1.352 1.352 0 00-.404-.957 1.378 1.378 0 00-.968-.396H5.783c.028-.088.056-.177.084-.255.274-.82 1.153-3.326 1.153-3.326a4.262 4.262 0 00-2.413.698c-.57.4-.912.682-1.371 1.946-.532 1.453-.997 2.856-1.31 3.693C1.444 8.674.28 11.025.28 11.025a5.85 5.85 0 002.52-.61c1.119-.593 1.679-1.502 2.054-2.883l.09-.3h2.334v5.5c0 .5-.045.982-.073 1.46h-4.12c-.71 0-1.39.278-1.893.775a2.638 2.638 0 00-.783 1.874h6.527a17.717 17.717 0 01-.778 3.649 16.796 16.796 0 01-3.012 5.273A33.104 33.104 0 010 28.74s3.13 1.175 5.425-.954c1.388-1.292 2.631-3.814 3.23-5.727a28.09 28.09 0 001.12-5.229h5.967v-1.37a1.254 1.254 0 00-.373-.899 1.279 1.279 0 00-.909-.37zm-3.19 5.484l-2.312 1.491 5.038 7.458a6.905 6.905 0 00.672-2.218 3.15 3.15 0 00-.28-2.168l-3.118-4.563zM29.05 4.582H16.733V25.94h3.018l.403 2.572 4.081-2.572h4.815V4.582zm-5.207 18.69l-2.396 1.509-.235-1.508h-1.724V7.233h6.78v16.04h-2.425z\" /></svg>"
+    },
+    {
+      "id": "voot",
+	    "name": "Voot",
```
Comment (Sergey-Lyapin) [inline assets/services.json:2151]: Please remove any extra spaces/tabs.

Look at the object above as an example.

**Review by Sergey-Lyapin — CHANGES_REQUESTED** [review]

File: assets/services.json (hunk 3 above)
Comment (gakudva) [reply assets/services.json]: Sorry for that oversight.  Now removed them.

**Comment by gakudva** [comment]
Sorry removed request for review by ameshkov and Alex-302 by error.  I am unable to undo this error.

**Review by Alex-302 — APPROVED** [review]

**Pull request merged**